SerialTerminal.hpp

```diff
@@ -183,6 +183,26 @@ namespace maschinendeck {
 
         return Pair<String, String>(keyword, message);
       }
+
+      static String ParseArgument(String message) {
```

Comment:
do you really think, this is neccesary? the cli gets split on the first space, what is the need for the quotes?

Reply:
yes, for example for password user-case.  Is working like a charm here:
https://github.com/hpsaturn/esp32-wifi-cli/

Reply:
I think that not, because is possible have passwords with spaces in the end. For example the next password: " xxx xxxx ". How you could handle it?